### PR TITLE
Create data migration rake task for encrypted dependents attrs

### PR DESCRIPTION
**Pull request by bytheway875**
Co-authored-by: Maria Quadri <mquadri@codeforamerica.org>

### CODE REVIEW

File: lib/tasks/backfill_encrypted_dependent_data.rake [hunk 1]
```diff
@@ -0,0 +1,13 @@
+namespace :dependents do
+  desc "Backfill account number to new encrypted column"
```
Comment (mquadri23) [inline lib/tasks/backfill_encrypted_dependent_data.rake:2]: sorry, could you remind me what the **account number** in this context was again?

**Review by mquadri23 — APPROVED** [review]
Looks good to me!

File: lib/tasks/backfill_encrypted_dependent_data.rake (hunk 1 above)
Comment (bytheway875) [reply lib/tasks/backfill_encrypted_dependent_data.rake:2]: ah, the description is not updated. good catch.

**Review by mquadri23 — APPROVED** [review]
Updated description great!

**Pull request merged**